modules/proc_use-quay-build-dockerfiles.adoc

```diff
@@ -204,43 +207,31 @@ Use the following procedure to create a Build trigger.
 
 . Select the desired platform, for example, Github, BitBucket, Gitlab, or use a custom Git repository. For this example, we are using Github.
 
-. If prompted, confirm access to your account.
-
-. When prompted, select an organization. You can filter namespaces by including text in the *Filter namespaces...* box. Alternatively, you can scroll your namespaces by clicking the directional arrows. If the organization that you are trying to select is not listed, click *Connections wih Quay Container Registry* to request, or grant yourself, access. 
-
-. Click *Continue* after you have selected an organization. 
-
-. When prompted, select a repository. Then, click *Continue*. 
-
-. Configure the trigger by selecting one of the following options:
-
-.. *Trigger for all branches and tags (default1)*. By selecting this option, a container image for each commit across all branches and tags is created. 
-
-.. *Trigger only on branches and tags matching a regular expression*. By selecting this option, only container images for a subset of branches and/or tags are built. 
-
-. Click *Continue*. 
+. Enter a repository name, for example, `\https://github.com/<username>/testrepo`. Then, click *Next*. 
```

Comment:
If we take `github` as example,  the enter page should be like this 
<img width="1169" alt="Screenshot 2024-03-14 at 15 51 36" src="https://github.com/quay/quay-docs/assets/58208478/b1c31fd9-77cf-4733-af38-b408e3a31fcc">
The github credentials have been configured in config.yaml ahead.  So customer doesn't need to input repository name. 
If we take `custom Git repository` as example,  now customer needs to input repository name,  but the format is `git@github.com:<username>/<repo>.git`.
<img width="1169" alt="Screenshot 2024-03-14 at 15 52 39" src="https://github.com/quay/quay-docs/assets/58208478/1d7d9457-1e78-4d9e-b927-2114e4c98447">

Reply:
@huweihua-redhat added the following information: 

[NOTE]
====
The following procedure assumes that you have not included Github credentials in your `config.yaml` file.
====

. Enter a custom Git repository name, for example, `\git@github.com:<username>/<repo>.git`. Then, click *Next*.